src/siembol/resources/storm-submit.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: batch/v1
```

Comment:
this is already in docker image of storm topology manager not needed

Reply:
'k, thanks

Reply:
@of13tch I wanted to check on this.  It still looks like `storm-submit.yaml` is used in the `templates/configmap-manager.yaml` file.  Do we really not need `storm-submit.yaml` in the `data` section of that template any longer?